src/init.cpp

```diff
@@ -1123,7 +1125,7 @@ bool AppInitParameterInteraction()
             for (int j=0; j<(int)Consensus::MAX_VERSION_BITS_DEPLOYMENTS; ++j)
             {
                 if (vDeploymentParams[0].compare(VersionBitsDeploymentInfo[j].name) == 0) {
-                    UpdateRegtestBIP9Parameters(Consensus::DeploymentPos(j), nStartTime, nTimeout);
+                    UpdateBIP9Parameters(Consensus::DeploymentPos(j), nStartTime, nTimeout);
```

Comment:
This impedes the function being called by non regtest-like chains: https://github.com/bitcoin/bitcoin/pull/8855/commits/f87f3626e35e7edae37e14f5156809325ad18cb2#diff-c865a8939105e6350a50af02766291b7R1107